core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -499,36 +502,56 @@ trait WorkspaceComponent {
     }
 
     /**
-     * gets the submission stats (last workflow failed date, last workflow success date, running submission count)
+     * gets the submission stats (last submission failed date, last submission success date, running submission count)
      * for each workspace
      *
      * @param workspaceIds the workspace ids to query for
      * @return WorkspaceSubmissionStats keyed by workspace id
      */
     def listSubmissionSummaryStats(workspaceIds: Seq[UUID]): ReadAction[Map[UUID, WorkspaceSubmissionStats]] = {
-      // workflow date query: select workspaceId, workflow.status, max(workflow.statusLastChangedDate) ... group by workspaceId, workflow.status
-      val workflowDatesQuery = for {
+
+      // submission date query:
+      //
+      // select workspaceId, status, max(submissionDate)
+      // from (
+      //   select distinct submission.workspaceId, workflow.status, submission.submissionDate
+      //   from submission
+      //   join workflow on workflow.submissionId = submission.id
+      //   where submission.workspaceId in (:workspaceIds)) v
+      // group by 1, 2
+      // having (status = 'Failure' or (status = 'Succeeded' and count(v.*) = 1))
```

Comment:
@helgridly based on our conversation how does this revised query look to you?
```
      select workspaceId, status, max(subEndDate)
      from (
        select submission.id, submission.workspaceId, workflow.status, max(workflow.statusLastChangedDate) as subEndDate
        from submission
        join workflow on workflow.submissionId = submission.id
        where submission.workspaceId in (:workspaceIds)
        group by 1, 2, 3) v
      group by 1, 2
      having (status = 'Failure' or (status = 'Succeeded' and count(v.id) = 1))
```
Explanation: 
- inner query returns the most recent workflow status change date, per workflow status
- outer query returns the most recent workflow status change date where:
  - the status is Failure; or
  - the status is Succeeded and that is the _only_ status in the submission

I can code this up and try to break it with tests too, just thought I'd post the SQL first.